submissions/aderonkeOdebunmi/introductionToHtml/index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
```

Comment:
@a-Ronke You already have a folder `IntroductionToHtml` and a `index.html`. Why are you adding a new folder and file?